geps/gep-713/index.md

```diff
@@ -1,1157 +1,1082 @@
-  Gateway API resources
-* Discuss the problems with communicating status for Policy objects, and suggest
-  mechanisms that Policy APIs can use to mitigate some of them.
-* Provide a way to distinguish between required and default values for all
-  policy API implementations
-* Enable policy attachment at all relevant scopes, including Gateways, Routes,
-  Backends, along with how values should flow across a hierarchy if necessary
-* Ensure the policy attachment specification is generic and forward thinking
-  enough that it could be easily adapted to other grouping mechanisms like
-  Namespaces in the future
-* Provide a means of attachment that works for both ingress and mesh
-  implementations of this API
-* Provide a consistent specification that will ensure familiarity between both
-  included and implementation-specific policies so they can both be interpreted
-  the same way.
-
-## Deferred Goals and Discussions
-
-* Should Policy objects be able to target more than one object? At the time of
-  writing, the answer to this is _no_, in the interests of managing complexity
-  in one change. But this rule can and should be discussed and reexamined in
-  light of community feedback that users _really_ want this. Any discussion will
-  need to consider the complexity tradeoffs here.
+* Establish a pattern for metaresources which will be used for any metaresources and policies included in the Gateway API spec.
+* Establish a pattern that must be adopted for any implementation-specific metaresources and policies used with Gateway API resources.
+* Discuss the problems with communicating status for metaresource objects, and suggest mechanisms that APIs can use to mitigate some of them.
+* Provide a way to distinguish between required and default values for all metaresource API implementations.
+* Enable Policy Attachment at all relevant scopes in Gateway API, including Gateways, Routes, Backends, along with how values should flow across a hierarchy if necessary.
+* Ensure the Policy Attachment specification is generic and forward thinking enough that it could be easily adapted to other grouping mechanisms like Namespaces in the future.
+* Provide a means of attachment that works for both ingress and mesh implementations of Gateway API.
+* Provide a consistent specification that will ensure familiarity between both included and implementation-specific metaresources so they can both be interpreted the same way.
+* Provide a reference pattern to other implementations of metaresource and policy APIs outside of Gateway API, that are based on similar concepts (i.e., augmenting the behavior of other Kubernetes objects, attachment points, nested contexts and inheritance, Defaults & Overrides, etc.)
 
 ## Out of scope
 
-* Define all potential policies that may be attached to resources
-* Design the full structure and configuration of policies
+* Define all potential metaresource and policy kinds that may be attached to resources.
+
+## Background
```

Comment:
Somewhere early in this GEP, I'd recommend adding some guidance on _when_ this pattern should be used. IE, if you can modify the API you're extending directly, that's often going to provide a better UX than policy attachment. I think you're saying something similar in this introduction, but it could be helpful to make that guidance more clear.

Reply:
I think what @robscott is saying is similar to the "prefer other means" @shaneutt mentioned in https://github.com/kubernetes-sigs/gateway-api/pull/3609/files?diff=split&w=0#r1971718148.

Reply:
I think this is better covered now with the risks and caveats called out at the top of the doc and the new "User stories" section.